test/credis_test.py

```diff
@@ -19,15 +23,19 @@ def tearDown(self):
         ray.worker.cleanup()
 
     def test_credis_started(self):
```

Comment:
What is the goal of this test suite eventually? Should we also test credis reads/writes here?

Reply:
I believe this file and this test method was created by @pcmoritz.  Here, I adhere to its original semantics to test credis is successfully started, even when this current PR changes to way the module is loaded.  In short, I intend to
* Use this Python test to test it connects to the front end.  (This is reflected in this PR; in fact this test method.)
* Use the C++ tests to test writes and reads.  (This is reflected in this PR.)

To address your general comment about testing: Travis's "RAY_USE_NEW_GCS=on" build performs an `export RAY_USE_NEW_GCS=on` at the beginning of that build, so everything related to (1) new GCS, _and_ (2) credis will be tested in that build.